jumpstart-launch-tasks.sh

```diff
@@ -15,9 +16,12 @@ drush @acsf.cardinald7.$sitename -y vset --always-set preprocess_css 1
 drush @acsf.cardinald7.$sitename -y vset --always-set preprocess_js 1
 drush @acsf.cardinald7.$sitename -y vset --always-set page_compression 0
 drush @acsf.cardinald7.$sitename -y vset --always-set redirect_auto_redirect 1
-drush @acsf.cardinald7.$sitename -y dis stanford_metatag_nobots nobots 
+drush @acsf.cardinald7.$sitename -y dis stanford_metatag_nobots nobots
 drush @acsf.cardinald7.$sitename upwd Howard --password="$howardpass"
 drush @acsf.cardinald7.$sitename upwd Lindsey --password="$lindseypass"
+if [ -n "`drush @acsf.cardinald7.$sitename sqlq 'SELECT * FROM users where name = "Richard"'`" ]; then
```

Comment:
Ah, nice. I wasn't sure that this extra query was necessary, but it looks like drush throws an exception if Richard does not exist, so I think the added overhead of the extra query to see if Richard exists is worth it for the improved UX of running this on non-JSE sites.